packages/zod-form-data/src/helpers.ts

```diff
@@ -42,6 +42,12 @@ export const checkbox = ({ trueValue = "on" }: CheckboxOpts = {}) =>
     z.literal(undefined).transform(() => false),
   ]);
 
+export const file: InputType<ZodTypeAny> = (schema = z.instanceof(File)) =>
```

Comment:
Another small thing I noticed. The default type is going to be from `instanceof` which appears to be typed this way.

```suggestion
export const file: InputType<z.ZodType<File, z.ZodTypeDef, File>> = (schema = z.instanceof(File)) =>
```